### PR TITLE
iOS READMEs - Geometry

### PR DESCRIPTION
**Pull request by yo1995**
Update README for `Geometry` category. See comments below.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Geometry/Spatial operations/README.md [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+Find the union, intersection, or difference of two geometries.
 
-## How to use the sample
+![Image of perform spatial operations](perform-spatial-operations.png)
+
+## Use case
+
+The different spatial operations (union, difference, symmetric difference, and intersection) can be used for a variety of spatial analyses. For example, government authorities may use the intersect operation to determine whether a proposed road cuts through a restricted piece of land such as a nature reserve or a private property.
+
+When these operations are chained together, they become even more powerful. An analysis of food deserts within an urban area might begin by union-ing service areas of grocery stores, farmer's markets, and food co-ops. Taking the difference between this single geometry of all services areas and that of a polygon delineating a neighborhood would reveal the areas within that neighborhood where access to healthy, whole foods may not exist.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Spatial operations/README.md:11]:
```suggestion
When these operations are chained together, they become even more powerful. An analysis of food deserts within an urban area might begin by union-ing service areas of grocery stores, farmers markets, and food co-ops. Taking the difference between this single geometry of all services areas and that of a polygon delineating a neighborhood would reveal the areas within that neighborhood where access to healthy, whole foods may not exist.
```

File: arcgis-ios-sdk-samples/Geometry/Clip geometry/README.md [hunk 2]
```diff
@@ -1,26 +1,34 @@
 # Clip Geometry
 
-This sample demonstrates how to clip a geometry with an envelope using `AGSGeometryEngine`.
+Clip a geometry with another geometry.
 
-![](image1.png)
+![Image of clip geometry](clip-geometry.png.png)
```
[the remark this answers is not in the document]
Comment (yo1995) [reply arcgis-ios-sdk-samples/Geometry/Clip geometry/README.md:5]: Ewww I missed that... 😓😓

[the remark this answers is not in the document]
Comment (philium) [reply arcgis-ios-sdk-samples/Geometry/Clip geometry/README.md:5]: That's why we do code reviews! =)

File: arcgis-ios-sdk-samples/Geometry/Buffer/README.md [hunk 3]
```diff
@@ -1,26 +1,39 @@
 # Buffer
 
-This sample demonstrates how to use `AGSGeometryEngine` to create polygons from a map location and linear distance (radius). For each input location, the sample creates two buffer polygons (using the same distance) and displays them on the map using different symbols. One polygon is calculated using the `planar` (projected) coordinate space of the map. The other is created using a `geodesic` technique that considers the curved shape of the Earth's surface. Distortion in the map increases as you move away from the standard parallels of the spatial reference's projection. This map uses the Web Mercator projection so areas near the equator are the most accurate. As you move the buffer location north or south from that latitude, you'll see a greater difference in the polygon size and shape. Planar operations are generally faster, but performance improvement may only be noticeable for large or complex buffer operations.
+Create a buffer around a map point and display the results as an `AGSGraphic`.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Geometry/Buffer/README.md:3]: You updated the description in `ContentPList.plist`, but not here. They should match.

```suggestion
Create a buffer around a map point and display the results as a graphic.
```

Comment (yo1995) [reply arcgis-ios-sdk-samples/Geometry/Buffer/README.md:3]: 🤦‍♂ Will change!

**Pull request merged**